components/faq-accordion.tsx

```
@@ -0,0 +1,99 @@
+                      height: { duration: 0.1 },
+                      opacity: { duration: 0.07 }
+                    }
+                  }}
+                  className="mt-[4px] text-base font-normal text-muted-foreground overflow-hidden"
+                >
+                  <motion.div
+                    initial={{ y: -10 }}
+                    animate={{ y: 0 }}
+                    exit={{ y: -10 }}
+                    transition={{ duration: 0.07 }}
+                    dangerouslySetInnerHTML={{ __html: item.answer }}
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)